sycl/source/handler.cpp

```diff
@@ -1292,19 +1329,31 @@ id<2> handler::computeFallbackKernelBounds(size_t Width, size_t Height) {
   return id<2>{std::min(ItemLimit[0], Height), std::min(ItemLimit[1], Width)};
 }
 
+#ifdef __INTEL_PREVIEW_BREAKING_CHANGES
+void handler::ext_intel_read_host_pipe(const detail::string_view &Name,
```

Comment:
```suggestion
void handler::ext_intel_read_host_pipe(detail::string_view Name,
```

Reply:
Sorry, I don't understand the rationale of this suggestion.